core/store/migrate/migrations/0227_add_server_url_to_transmit_requests.sql

```diff
@@ -0,0 +1,9 @@
+-- +goose Up
+ALTER TABLE mercury_transmit_requests DROP CONSTRAINT mercury_transmit_requests_pkey;
+DELETE FROM mercury_transmit_requests;
+ALTER TABLE mercury_transmit_requests ADD COLUMN server_url TEXT NOT NULL;
+ALTER TABLE mercury_transmit_requests ADD PRIMARY KEY (server_url, payload_hash);
+
+-- +goose Down
+ALTER TABLE mercury_transmit_requests DROP COLUMN server_url;
+ALTER TABLE mercury_transmit_requests ADD PRIMARY KEY (payload_hash);
```

Comment:
do you not need to drop the pkey constraint again `mercury_transmit_requests_pkey` before creating the pkey with `payload_hash`?

Reply:
Will be autodropped along with the column it depends on (server_url in this case)